s32/drivers/s32k1/BaseNXP/header/S32K146_ADC.h

```diff
@@ -1,6 +1,6 @@
 /*
  * Copyright 1997-2016 Freescale Semiconductor, Inc.
- * Copyright 2016-2022 NXP
+ * Copyright 2016-2024 NXP
```

Comment:
pelase do not change any copyright notice that is not yours/your organization, you can add yours below if required

Reply:
ok